server/src/models/types.ts

```diff
@@ -0,0 +1,245 @@
+export type entity = userModel | addressModel | countryModel | stateModel | cityModel | cartModel | number | string;
+
+interface loginDetail {
```

Comment:
Naming convention for interface
Generally prefix "I" is added before an interface + Pascal case